src/hs_input_plugins.c

```diff
@@ -300,6 +300,18 @@ create_input_plugin(const hs_config *cfg, hs_sandbox_config *sbc)
 }
 
 
+static struct timespec get_current_timespec(const char *name)
+{
+  struct timespec ts;
+  if (clock_gettime(CLOCK_REALTIME, &ts) == -1) {
```

Comment:
should we use monotonic clock here?

Reply:
No, functions like sem_timedwait and pthread_timedjoin_np need the real time value.